### PR TITLE
Feat/ingredient delete

### PR DESCRIPTION
**Pull request by shaySirek**
delete ingredient implementation

### CODE REVIEW

**Comment by shaySirek** [comment]
[Ingredient delete implementation](https://trello.com/c/Nx0zpsEj/34-ingredient-delete-implementation)

File: Controllers/IngredientController.cs [hunk 1]
```diff
@@ -125,20 +125,19 @@ public async Task<ActionResult> Delete(int? id)
...
-                db.RecipeIngredients.Any(ri => ri.IngredientID == id && ri.RecipeID == r.ID)).ToList();
-            var recipeController = new RecipeController();
-
-            recipesUsingRemovedIngredient.ForEach(recipe => recipeController.DeleteRecipe(recipe.ID));
-            db.Ingredients.Remove(ingredient);
-            await db.SaveChangesAsync();
-            return RedirectToAction("Index");
-        }*/
+            Ingredient ingredient = await this.bl.GetIngredientById(id);
+            if (ModelState.IsValid)
```
Comment (RanNissan) [inline Controllers/IngredientController.cs:134]: What is this?

Comment (shaySirek) [reply Controllers/IngredientController.cs:134]: get ingredient for deletion

Comment (shaySirek) [reply Controllers/IngredientController.cs:134]: should I send the id to Delete method instead of sending ingredient object?

Comment (RanNissan) [reply Controllers/IngredientController.cs:134]: I asked about the if statement..
anyway, getting by id is preferable..

**Pull request merged**